code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/integration/ldap/OpenLDAP/OpenLDAPIdentityProvider.java

```diff
@@ -77,13 +78,19 @@
 
     private Set<Identity> getIdentities(LdapName dn) {
         Set<Identity> identities = new HashSet<>();
-        Attributes userAttributes;
+        Attributes userAttributes, opAttributes = null;
         LdapContext context = getServiceContext();
         try {
             userAttributes = context.getAttributes(dn);
             if (!hasPermission(userAttributes)){
                 throw new ClientVisibleException(ResponseCodes.UNAUTHORIZED);
             }
+            try {
+                String[] operationalAttrList = {"1.1", "+", "*"};
```

Comment:
http://www.openldap.org/lists/openldap-technical/201103/msg00296.html
talks about this ldapsearch attribute list to list all operational attributes.

Also http://linuxcommand.org/man_pages/ldapsearch1.html  
explains the "+" and "*":
If * is listed, all user attributes are returned.
       If + is listed, all operational attributes are returned.  If  no  attrs
       are  listed,  all user attributes are returned.  If only 1.1 is listed,
       no attributes will be returned